BUILD.gn

```diff
@@ -165,23 +165,22 @@ run_node("bundle") {
     "js/os.ts",
     "js/runtime.ts",
     "js/types.ts",
-    "js/util.ts",
-    "package.json",  # The `browserslist` field controls Babel behavior.
+    "js/util.ts"
   ]
   outputs = [
     out_dir + "main.js",
   ]
   deps = [
-    ":flatbufferjs",
-    ":run_tsc",
+    ":flatbufferjs"
   ]
   args = [
-    "./node_modules/parcel-bundler/bin/cli.js",
-    "build",
-    "--no-minify",
-    "--out-dir",
-    rebase_path(out_dir, root_build_dir),
+    "./node_modules/rollup/bin/rollup",
+    "-c",
+    rebase_path("third_party/rollup.config.js", root_build_dir),
```

Comment:
`rollup.config.js` is not in third_party dir.
```patch
-    rebase_path("third_party/rollup.config.js", root_build_dir),
+    rebase_path("rollup.config.js", root_build_dir),
```

Reply:
It is because the `BUILD.gn` copies it there. Because other meta config gets copied there, I assumed that was the preferred pattern.

Reply:
Yeah, I see, but something is not right about paths, in [`build_third_party.py`](https://github.com/ry/deno/pull/368/files#diff-485d7aa6a3223f30e699c54eec70a2ffR25) you create a symbolic link to `rollup.config.js` inside `third_party` dir, and then here you suppose `rollup.config.js` is inside third_party dir, but in `rollup.config.js` you have [this line](https://github.com/ry/deno/pull/368/files#diff-ff6e5f22a9c7e66987b19c0199636480R18) which has this `third_party/` prefix before the path.

Reply:
While that might be excessive indirection, it doesn't appear to be related to the issue I am experiencing.  Changing `BUILD.gn` to point at the root `rollup.config.js` and removing the linking in `third_party` does not resolve the issue, the behaviour is the same.  When the CWD is `out/Debug` it fails, when the CWD is the root, it works.